src/app/task-form-dialog/task-form-dialog.component.ts

```diff
@@ -53,4 +53,17 @@ export class TaskFormDialogComponent implements OnInit {
       )
     }
   }
+  titleValidator(control:FormControl)
+  {
+    const exp:RegExp=/^[A-Z]\w+/gm;
+    
+    if(control.value!=null&&!exp.test(control.value))
```

Comment:
plz use strict inequality
also this if-else can be simplified with the ternary operator